app/interfaces/models.ts

```diff
@@ -47,4 +47,14 @@ export interface IKey {
   user_id: string;
   created_at?: Date;
   updated_at?: Date;
+}
+
+export interface IWallet {
+  id?: number;
+  app_id: string;
```

Comment:
When you update the model, update the interfaces too. Also, add interfaces for other models you added